Source/Immutable/Public/Immutable/ImmutablePassport.h

```diff
@@ -503,6 +513,21 @@ class IMMUTABLE_API UImmutablePassport : public UObject {
   ImxBatchNftTransfer(const FImxBatchNftTransferRequest &RequestData,
                       const FImtblPassportResponseDelegate &ResponseDelegate);
 
+  /**
+   *
+   * Checks if a passport registered off-chain.
+   * @param ResponseDelegate The response delegate of type
+   * FImtblPassportResponseDelegate to call on response from JS.
+   */
+  void ImxIsRegisteredOffchain(const FImtblPassportResponseDelegate& ResponseDelegate);
+  
+  /**
+   * Registers a passport off-chain.
```

Comment:
```suggestion
   * Register the user to Immutable X if they are not already registered
```

I just adapted it from [here](https://docs.immutable.com/docs/x/passport/imx-provider/#registeroffchain)